scripts/addCandidateBalances.js

```diff
@@ -12,7 +12,6 @@ async function main() {
   assert(accounts != null && Object.getPrototypeOf(accounts) === Object.prototype);
   for (const candidate of constants.CANDIDATES) {
     assert(!Object.prototype.hasOwnProperty.call(accounts, candidate.staking));
-    accounts[candidate.staking] = { balance: '0x100000000000000000' };
```

Comment:
Is this file required at all now?

Reply:
No, it is not.

Reply:
Can we just remove this file (`scripts/addCandidateBalances.js`) altogether and do `cp posdao-contracts/spec.json parity-data/` instead?

Reply:
Yes, we can.